gulpfile.babel.js

```diff
@@ -9,7 +9,7 @@ const plugins = loadPlugins({
   lazy: false,
 });
 
-import babelRegister from 'babel/register';
+import babelRegister from 'babel-core/register';
```

Comment:
[babel docs](https://babeljs.io/docs/usage/require/) suggest otherwise, any idea where the disconnect comes from?